colcon_cmake/task/cmake/__init__.py

```diff
@@ -206,6 +207,20 @@ def get_project_file(path, target):
     return project_file
 
 
+@functools.lru_cache(maxsize=1)
+def get_msbuild_version():
+    """
+    Get the version of msbuild or throw an exception.
+
+    :rtype: str
+    """
+    completed_process = subprocess.run(
```

Comment:
Please invoke `check_output` instead of `run` since that is what it is used for here.